src/cosima/inc/MCSteppingAction.hh

```diff
@@ -87,7 +87,8 @@ public:
   static const int c_ProcessIDIonization;
   static const int c_ProcessIDTransportation;
   static const int c_ProcessIDUncovered;
-  
+  static const int c_ProcessIDNoProcess; //added in g4.11 : New class G4NoProcess, representing an empty process not assigned to any particle, and used only in G4SteppingManager::InvokeAtRestDoItProcs() for stable ions at rest, in order to avoid that radioactive decay appears as process defining their last step. Note that the method IsApplicable(), retunrs false, as this process should not be set to any particle. (see release note)
+
```

Comment:
now at the end of each process there is a `NoProcess` ID .
Example : 
```
G4WT0 > Step#      X         Y         Z        KineE     Velocity   dEStep  StepLeng  TrakLeng       Volume     Process
G4WT0 >     0  -1.57 m     -16 cm  -14.6 cm   1.21 keV  2.06 cm/ns     0 eV      0 fm      0 fm        World    initStep
G4WT0 >     1  -1.57 m     -16 cm  -14.6 cm   1.05 keV  2.06 cm/ns   159 eV   5.67 um   5.67 um        World         msc
G4WT0 >     2  -1.57 m     -16 cm  -14.6 cm    806 eV   1.92 cm/ns   242 eV   26.1 um   31.7 um        World       eIoni
G4WT0 >     3  -1.57 m     -16 cm  -14.6 cm    491 eV   1.68 cm/ns   314 eV   21.8 um   53.5 um        World       eIoni
G4WT0 >     4  -1.57 m     -16 cm  -14.6 cm      0 eV   1.31 cm/ns   491 eV   16.2 um   69.7 um        World       eIoni
G4WT0 >     5  -1.57 m     -16 cm  -14.6 cm      0 eV      0 mm/s      0 eV      0 fm   69.7 um        World   NoProcess
```
So i added a `c_ProcessIDNoProcess` variable in order to not trigger the `Uncovered process` error of MEGAlib